controller/planner_backlog.go

```diff
@@ -93,10 +93,11 @@ func generateBacklogExpression(ctx context.Context, db application.DB, spaceID u
 
 		// Get the list of work item types that derive of PlannerItem in the space
 		var expWits criteria.Expression
-		wits, err := appl.WorkItemTypes().ListPlannerItems(ctx, spaceID)
+		witsOrig, err := appl.WorkItemTypes().ListPlannerItems(ctx, spaceID)
 		if err != nil {
 			return errs.Wrap(err, "unable to fetch work item types that derive from planner item")
 		}
+		wits := stripBaseWorkItemTypes(witsOrig)
```

Comment:
The path of the planner item is the empty one so it won't be included in the returned list. So there isn't necessary to do this filtering.